src/python/pants/backend/go/target_types.py

```diff
@@ -151,6 +152,13 @@ class GoPackageSourcesField(MultipleSourcesField):
     default = ("*.go", "*.s")
     expected_file_extensions = (".go", ".s")
     ban_subdirectories = True
+    help = generate_multiple_sources_field_help_message(
+        softwrap(
+            """
+            Example: `sources=['example.go', '*_test.go', '!test_ignore.go']`.
+            """
+        )
```

Comment:
No need to softwrap because this fits on one line:

```suggestion
        "Example: `sources=['example.go', '*_test.go', '!test_ignore.go']`"
```

I also got rid of the period

(Ditto all other messages)

Reply:
Done!